Clean up the Cloud Console roles topic (#19758)
cockroachdb/docs: src/current/_includes/cockroachcloud/cluster-operator-prereq.md

> Either the [Cluster Operator]({% link cockroachcloud/authorization.md %}#cluster-operator) or [Cluster Admin]({% link cockroachcloud/authorization.md %}#cluster-admin) role on a pre-existing cluster, or the [Cluster Creator](authorization.html#cluster-creator) role in order to create a new cluster.

Reviewer: ```suggestion
Either the [Cluster Operator]({% link cockroachcloud/authorization.md %}#cluster-operator) or [Cluster Admin]({% link cockroachcloud/authorization.md %}#cluster-admin) role on an existing cluster, or the [Cluster Creator](authorization.html#cluster-creator) role to create a new cluster.
```
